core-common/src/test/java/org/zalando/nakadi/validation/ISO4217CurrencyCodeValidatorTest.java

```diff
@@ -31,10 +32,14 @@ public void testValidator() {
                 "SEK",
         };
 
-        final var validator = new ISO4217CurrencyCodeValidator("an-event-type");
+        final var validator = new ISO4217CurrencyCodeValidator();
         for (final String value : invalidValues) {
             // For invalid values, the validator should return empty result.
```

Comment:
Looks like the comment should be removed now?

Reply:
I've missed that line.